src/NServiceBus.Testing/Saga.cs

```diff
@@ -1,11 +1,11 @@
 ﻿namespace NServiceBus.Testing
 {
     using System;
+    using System.Collections.Generic;
     using System.Linq;
-    using System.Reflection;
     using System.Threading.Tasks;
-    using NServiceBus.MessageInterfaces.MessageMapper.Reflection;
-    using NServiceBus.Testing.ExpectedInvocations;
+    using MessageInterfaces.MessageMapper.Reflection;
+    using ExpectedInvocations;
```

Comment:
Nothing that needs to be addressed in this PR but I just realized that the namespace for the invocations probably doesn't add much. Could we consider ditching it?